packages/field/src/Field.tsx

```diff
@@ -0,0 +1,229 @@
+  | 'assetGalleryEditor'
+  | 'richTextEditor'
+  | 'markdown'
+  | string;
+
+type FieldProps = {
+  sdk: FieldExtensionSDK;
+  widgetId: WidgetType;
+  isInitiallyDisabled: boolean;
+  renderFieldEditor?: (
+    widgetId: WidgetType,
+    sdk: FieldExtensionSDK,
+    isInitiallyDisabled: boolean
+  ) => JSX.Element | false;
+};
+
+export const Field: React.FC<FieldProps> = ({
+  sdk,
+  widgetId,
+  isInitiallyDisabled,
+  renderFieldEditor,
+}: FieldProps) => {
+  const field = sdk.field;
+  const locales = sdk.locales;
+
+  if (renderFieldEditor) {
+    const customEditor = renderFieldEditor(widgetId, sdk, isInitiallyDisabled);
+    if (customEditor) {
+      return customEditor;
+    }
+  }
+
+  switch (widgetId) {
+    case 'multipleLine':
+      return (
+        <MultipleLineEditor
+          field={field}
+          locales={locales}
+          isInitiallyDisabled={isInitiallyDisabled}
+        />
+      );
+    case 'boolean':
+      return <BooleanEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'objectEditor':
+      return <JsonEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'datePicker':
+      return <DateEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'locationEditor':
+      return <LocationEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'checkbox':
+      return (
+        <CheckboxEditor field={field} locales={locales} isInitiallyDisabled={isInitiallyDisabled} />
+      );
+    case 'listInput':
+      return (
+        <ListEditor field={field} locales={locales} isInitiallyDisabled={isInitiallyDisabled} />
+      );
+    case 'rating':
+      return <RatingEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'radio':
+      return (
+        <RadioEditor field={field} locales={locales} isInitiallyDisabled={isInitiallyDisabled} />
+      );
+    case 'tagEditor':
+      return <TagsEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'numberEditor':
+      return <NumberEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'urlEditor':
+      return <UrlEditor field={field} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'slugEditor':
+      return <SlugEditor field={field} baseSdk={sdk} isInitiallyDisabled={isInitiallyDisabled} />;
+    case 'singleLine':
+      return (
+        <SingleLineEditor
+          field={field}
+          locales={locales}
+          isInitiallyDisabled={isInitiallyDisabled}
+        />
+      );
+    case 'dropdown':
+      return (
+        <DropdownEditor field={field} locales={locales} isInitiallyDisabled={isInitiallyDisabled} />
+      );
+    case 'entryLinkEditor': {
+      return (
+        <SingleEntryReferenceEditor
+          viewType="link"
+          sdk={sdk}
+          isInitiallyDisabled={isInitiallyDisabled}
+          hasCardEditActions={true}
+          parameters={{
```

Comment:
This should be `sdk.parameters` by default and should fall back to these default values in case if `sdk.parameters.instance` are missing